corehq/apps/locations/permissions.py

```diff
@@ -164,12 +164,24 @@ class LocationResource(HqBaseResource):
     return resource
 
 
+def conditionally_location_safe(conditional_function):
+    """Decorator to apply to a view function that verifies if something is location
+    safe based on the arguments or kwarguments. That function should return
+    True or False.
+
+    """
+    def _inner(view_fn):
+        view_fn._conditionally_location_safe_function = conditional_function
+        return view_fn
+    return _inner
+
+
 def location_restricted_response(request):
     from corehq.apps.hqwebapp.views import no_permissions
     return no_permissions(request, message=LOCATION_ACCESS_DENIED)
 
 
-def is_location_safe(view_fn, view_args, view_kwargs):
+def is_location_safe(view_fn, view_args, view_kwargs, request=None):
```

Comment:
I can't find any places where this new `request` parameter is used by implementations of `_conditionally_location_safe_function`. Is it needed?